docs-site/content/0.19.0/guide/features/faceting.md

```diff
@@ -1 +1,215 @@
 # Faceting
+
+Faceting is a way to classify results in smaller groups. It helps users to narrow down their search based on certain parameters. For example, in a dataset of books, users may want to search for books by categorizing them first via author, genre, etc. Similarly, in a movie dataset, users may want to search for movies by first grouping them via actor, director, genre, etc. This subclassification of search results is known as Faceting. Faceting is not to be confused with Filtering, as Filtering provides a way to remove results that don't meet certain criteria, while Faceting provides a way to subgroup documents that can be searched upon further.
```

Comment:
> Faceting is a way to classify results in smaller groups. It helps users to narrow down their search based on certain parameters.

Faceting is a way to classify search results into smaller groups based on the attributes in each result. 

> For example, in a dataset of books, users may want to search for books by categorizing them first via author, genre, etc.

For example, in a dataset of books, after users search for a book title, they'd want to categorize the search results by authors, genre, etc. 

> Faceting is not to be confused with Filtering, as Filtering provides a way to remove results that don't meet certain criteria, while Faceting provides a way to subgroup documents that can be searched upon further.

Faceting is not to be confused with Filtering, as Filtering provides a way to remove results that don't meet certain criteria, while Faceting provides a way to group results into categories